wd_util.c

```diff
@@ -2233,7 +2233,7 @@ void *wd_dlopen_drv(const char *cust_lib_dir)
 		if (ret < 0)
 			goto free_node;
 
-		node->dlhandle = dlopen(lib_path, RTLD_NOW);
+		node->dlhandle = dlopen(lib_path, RTLD_NODELETE | RTLD_NOW);
```

Comment:
After modifying the problem of dlopen, please modify the supporting dlclose together:
Line 1998: dlclose(dlnode->dlhandle);

Reply:
[dae358e](https://github.com/Linaro/uadk/pull/552/commits/dae358ecb5ed5ca82fe138e86081a79dfee1c111)